flang/runtime/CMakeLists.txt

```diff
@@ -58,3 +57,7 @@ add_flang_library(FortranRuntime
   LINK_LIBS
   FortranDecimal
 )
+
+add_flang_library(Fortran_main STATIC PARTIAL_SOURCES_INTENDED 
```

Comment:
Broke f18-driver.  Why was Fortran_main removed from the library?

Reply:
> Broke f18-driver. Why was Fortran_main removed from the library?

I fixed f18-driver.

Reply:
We needed to separate main and runtime for the shared library builds. 
Why are the drivers separate?